tests/simple/simple_test.go

```diff
@@ -119,6 +136,45 @@ func TestSimpleFile(t *testing.T) {
 	if rc == nil {
 		return
 	}
+	var skipTests []string
+	firstFlag := true
+	for _, flagVal := range skipFlags {
+		fileInd := strings.Index(flagVal, "/")
+		splitFile = strings.SplitN(flagVal, "/", 2)
+		fileName := splitFile[0]
+		sectionString := splitFile[1]
+		if fileInd > 0 || sectionString == "" {
+			log.Fatal("skip_test argument must contain at least <file>/<section>, received ", flagVal)
+		}
+		for _, sectionVal := range strings.Split(sectionString, ";") {
+			sections := strings.Count(sectionVal, "/")
+			if sections == 0 {
+				if sectionVal != "" {
+					skipTests = append(skipTests, filename+"/"+sectionVal)
+				} else {
+					log.Fatal("Empty string where should be section name")
+				}
+			} else if sections == 1 {
+				lastInd := strings.LastIndex(sectionVal, "/")
+				sectionName := sectionVal[:lastInd+1]
+				testString := sectionVal[lastInd+1:]
+				if testString == "" {
+					log.Fatal("Empty string where should be test name")
+				} else {
+					tests := strings.Split(testString, ",")
+					for _, test := range tests {
+						if test == "" {
+							log.Fatal("Empty string where should be test name")
+						} else {
+							skipTests = append(skipTests, filename+"/"+sectionName+test)
```

Comment:
Here's why we want the consistency:  this sure looks like a bug.  It should either have two slashes or zero.